bundles/mapping/mapmodule/MapModuleClass.ol.js

```diff
@@ -1114,14 +1165,22 @@ export default class MapModule extends AbstractMapModule {
     }
 
     getGeoJSONGeometryFromOL (feature) {
-        var olGeoJSON = new olFormatGeoJSON();
-        var geojsonStr = olGeoJSON.writeFeature(feature);
-        var geojson = JSON.parse(geojsonStr);
-        if (geojson.geometry) {
-            return geojson.geometry;
+        var geojson = this.getGeoJSONFromFeatures([feature]);
+        if (geojson.features & geojson.features.length > 0) {
+            return geojson.features[0].geometry;
         }
         return null;
     }
+    /**
+     * @method getGeoJSONFromFeatures
+     * @param {Array] features
+     * @return {Object} geojson FeatureCollection
+     */
+    getGeoJSONFromFeatures (features) {
+        var olGeoJSON = new olFormatGeoJSON();
+        var geojsonStr = olGeoJSON.writeFeatures(features);
```

Comment:
`writeFeaturesObject` returns the GeoJSON as JS object. Using it we would not need to parse the string on the net line.